packages/oas-validator/index.js

```diff
@@ -13,17 +13,16 @@ let ajv = require('ajv')({
     allErrors: true,
     verbose: true,
     jsonPointers: true,
-    patternGroups: true,
```

Comment:
This option was removed on https://github.com/ajv-validator/ajv/commit/e15f543dbda7fc539817f2b9b2820f6db2e3986f